plugins/modules/web_infrastructure/django_manage.py

```diff
@@ -13,40 +13,41 @@
 module: django_manage
 short_description: Manages a Django application.
 description:
-    - Manages a Django application using the I(manage.py) application frontend to I(django-admin). With the I(virtualenv) parameter, all
-      management commands will be executed by the given I(virtualenv) installation.
+    - Manages a Django application using the C(manage.py) application frontend to C(django-admin). With the
+      C(virtualenv) parameter, all management commands will be executed by the given C(virtualenv) installation.
 options:
   command:
-    choices: [ 'cleanup', 'collectstatic', 'flush', 'loaddata', 'migrate', 'runfcgi', 'syncdb', 'test', 'validate', ]
+    choices: [ 'cleanup', 'collectstatic', 'flush', 'loaddata', 'migrate', 'runfcgi', 'syncdb', 'test', 'validate' ]
     description:
-      - The name of the Django management command to run. Built in commands are cleanup, collectstatic, flush, loaddata, migrate, runfcgi, syncdb,
-        test, and validate.
-      - Other commands can be entered, but will fail if they're unknown to Django.  Other commands that may prompt for user input should be run
-        with the I(--noinput) flag.
+      - The name of the Django management command to run. Built in commands are C(cleanup), C(collectstatic),
+        C(flush), C(loaddata), C(migrate), C(runfcgi), C(syncdb), C(test), and C(validate).
+      - Other commands can be entered, but will fail if they're unknown to Django.  Other commands that may
+        prompt for user input should be run with the I(--noinput) flag.
```

Comment:
```suggestion
        prompt for user input should be run with the C(--noinput) flag.
```
I(...) should be only used for module options.